recipes-devtools/python/python3-janus_0.6.1.bb

```diff
@@ -0,0 +1,21 @@
+DESCRIPTION = "Mixed sync-async queue to interoperate between asyncio tasks and classic threads"
+HOMEPAGE = "https://github.com/aio-libs/janus/"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=23878c357ebb4c8ce1109be365043349"
+
+RDEPENDS_${PN} = " \
+"
+
+SRC_URI = " \
+    git://github.com/aio-libs/janus.git;branch=master \
+    "
+
+# Version 0.6.1
+SRCREV = "9fb1ae5dc7bb503d54e7dffd73f157a9eb4009d1"
```

Comment:
Maybe a pypi recipe is even shorter? see https://github.com/intel-iot-devkit/meta-iot-cloud/blob/master/recipes-devtools/python/python3-janus_0.4.0.bb